_events_upcoming/2018-05-03-8th-pint-workshop.md

```diff
@@ -23,4 +23,12 @@ no_lead: true
 
 The Eighth Parallel-in-time Integration Workshop will be held from 20 to 24 May 2019 at the [Institute for Advanced Studies](http://www.uni-bielefeld.de/(en)/ZiF/Allgemeines/index.html) in [Bielefeld, Germany](https://goo.gl/maps/YiSDjkzd7qq).
 
-More information will follow in due time.
+We gladly confirmed the following invited speakers:
+
+ - Dr Carmen Rodrigo, University of Zaragosa
```

Comment:
This is Zaragoza, it seems..

Reply:
Other than that: great!

Reply:
It is indeed, pardon my French, eh, Spanish.